code/game/area/areas/ruins/beachplanet.dm

```diff
@@ -69,3 +69,16 @@
 /area/ruin/beach/treasure_cove
 	name = "Pirate Cavern"
 	icon_state = "purple"
+
+//beach_float_resort --> keeping resort open for a land based ruin
+
+/area/ruin/beach/float_resort
+	name = "Beach Resort"
+	icon_state = "yellow"
+	mood_bonus = 10
+	mood_message = "<span class='nicegreen''>Incredible! A hidden resort all to myself and my crew! </span>\n"
```

Comment:
this is a neat idea, but - please do not do this! moodlets tend to detract from RP.